master/buildbot/reporters/github.py

```diff
@@ -135,12 +142,24 @@ def send(self, build):
         else:
             issue = None
 
-        if project:
+        if "/" in project:
             repoOwner, repoName = project.split('/')
-        else:
+        # Given we have a repository define try to get the owner and name
+        elif sourcestamps[0]['repository']:
```

Comment:
Maybe using the `project` property is safe since github changes coming in via the www hook or `PullrequestPoller` have the project property set to the correct value only timed/force schedulers would have to set that property.